CONTRIBUTING.md

```diff
@@ -30,6 +30,8 @@ Just make sure to run the whole test suite before submitting a pull request!
 
 ### Pull Requests
 
+**Working on your first Pull Request?** You can learn how from this *free* series [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github)
```

Comment:
Full disclosure. The resource itself is absolutely free and always will be. However I **do** make money when people watch it. Thought it appropriate to call that out.